### PR TITLE
BZ2078187 Fix: VMware minimal privilege set is not included in MTV 2.3

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
BZ2078187 Fix: VMware minimal privilege set is not included in MTV 2.3

https://bugzilla.redhat.com/show_bug.cgi?id=2078187

MTV 2.3

### CODE REVIEW

File: documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc [hunk 1]
```diff
@@ -122,13 +124,13 @@ include::modules/canceling-migration-ui.adoc[leveloffset=+2]
 [id="migrating-virtual-machines-from-cli"]
 == Migrating virtual machines from the command line
 
-You can migrate virtual machines to {virt} from the command line.
+You can migrate virtual machines to {virt} from the command line. xref:../../../modules/vmware-prerequisites.adoc#vmware-privileges[VMware privileges]
```
Comment (mguetta1) [inline documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:127]: @sbeskin-redhat what is this additional link? looks like it is broken

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Comment by mguetta1** [comment]
@sbeskin-redhat please apply also the changes made here: https://github.com/konveyor/forklift-documentation/pull/282/files#diff-34cd0c83438a7adb7c57b031ab1c68de180e668e14dc882e26f4b37cee901f76L52

File: documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc]: @mguetta1 
Removed, thank you.
Please approve

**Review by mguetta1 — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**